google/cloud/bigtable/client.py

```diff
@@ -315,15 +325,28 @@ def __init__(
                 instance_id is combined with the client's project to fully
                 specify the instance
             table_id: The ID of the table.
-            app_profile_id: (Optional) The app profile to associate with requests.
+            app_profile_id: The app profile to associate with requests.
```

Comment:
isnt app profiel also optional?

Reply:
Yup, fixed. 

In general, we don't always show this in the docstrings if using type annotations in the signature, but I see timeouts have Optionals listed, so I made the change here too